pkg/minikube/machine/start.go

```diff
@@ -218,17 +219,32 @@ func postStartValidations(h *host.Host, drvName string) {
 		glog.Warningf("error getting command runner: %v", err)
 	}
 
+	var kind reason.Kind
+	var name string
+	if drvName == oci.Docker {
+		kind = reason.RsrcInsufficientDockerStorage
+		name = "Docker"
```

Comment:
better to use this helper instead
```
driver.FullName(drvName)
```

Reply:
But that one has some weird "Docker Desktop" special case ?

I agree it is not ideal to compare driver name and program name